app/api/v2/models/auth_models.py

```diff
@@ -38,10 +38,9 @@ def signup(self, firstname=None, lastname=None, othername=None, email=None, \
         query = """INSERT INTO users (firstname, lastname,\
          othername, email, phoneNumber, username, isAdmin, password) \
         VALUES (%(firstname)s, %(lastname)s, %(othername)s, %(email)s, %(phoneNumber)s, %(username)s, \
-        %(isAdmin)s, %(password)s) RETURNING user_id"""
+        %(isAdmin)s, %(password)s) RETURNING user"""
```

Comment:
I think you should return a valid users table field rather than user.

Reply:
@bencyn adjusted accordingly. Thank you.